tests/EntryPoints/Rest/SaveMappingsApiTest.php

```diff
@@ -569,4 +577,8 @@ private function createEmptyObjectBody(): string {
 		]';
 	}
 
+	private function setRequestUser( User $user ): void {
```

Comment:
I changed this test to use the `User` object instead of the `Authority`. Not sure if there are other implications, but we need the `User` object later to check authorization and it's not possible to get the `User` from an `Authority` (as far as I can tell).